packages/website/components/page-templates/package-home.js

```diff
@@ -48,26 +50,42 @@ Header.propTypes = {
   heading: PropTypes.string.isRequired,
 };
 
-const PackageHome = ({ data, children }) => (
-  <NavigationWrapper
-    navContent={() => (
-      <PackageNavContent packageId={data.id} packageName={data.packageName} />
-    )}
+const MissingReadmeBanner = () => (
+  <Banner
+    icon={<WarningIcon label="Warning icon" secondaryColor="inherit" />}
+    isOpen
   >
-    <Wrapper>
-      <Header id={data.id} heading={titleCase(data.packageName)} />
-      <Description>{data.description}</Description>
-      <PackageMetaData
-        id={data.id}
-        version={data.version}
-        maintainers={data.maintainers}
-        repository={data.repository}
-      />
-      {children}
-    </Wrapper>
-  </NavigationWrapper>
+    There is no README for this package. This warning is only visible in dev
+    mode.
+  </Banner>
 );
 
+const PackageHome = ({ data, children }) => {
+  return (
+    <NavigationWrapper
+      navContent={() => (
+        <PackageNavContent packageId={data.id} packageName={data.packageName} />
+      )}
+    >
+      <Wrapper>
+        <Header id={data.id} heading={titleCase(data.packageName)} />
+        <Description>{data.description}</Description>
+        <PackageMetaData
+          id={data.id}
+          version={data.version}
+          maintainers={data.maintainers}
+          repository={data.repository}
+        />
+        {children || process.env.NODE_ENV !== 'development' ? (
```

Comment:
Does this work when the component is rendered client-side?

Reply:
Seems like it

Reply:
...how? there shouldn't be a `process` object on the browser